tests/integration/test_vulnerability_detector/test_feeds/test_invalid_type_custom_feeds.py

```diff
@@ -0,0 +1,131 @@
+# Copyright (C) 2015-2020, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import pytest
+import os
+import tempfile
+
+from wazuh_testing.tools import LOG_FILE_PATH
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.tools import file
+import wazuh_testing.vulnerability_detector as vd
```

Comment:
```suggestion
import pytest
import os
import tempfile
import wazuh_testing.vulnerability_detector as vd

from wazuh_testing.tools import file, LOG_FILE_PATH
from wazuh_testing.tools.configuration import load_wazuh_configurations
from wazuh_testing.tools.monitoring import FileMonitor
```

Reply:
Done in ba7bcc0729d9649d5d646f0d5ea80a440267f65b

Reply:
Not for this file.

Reply:
Done in ca26474b64f76ba719b585d27df0f1728233f213